packages/orm/src/EntityManager.ts

```diff
@@ -745,6 +745,7 @@ export class EntityManager<C extends HasKnex = HasKnex> {
   async refresh(entities: ReadonlyArray<Entity>): Promise<void>;
   async refresh(entityOrListOrUndefined?: Entity | ReadonlyArray<Entity>): Promise<void> {
     this.findLoaders = {};
+    this.__data.loaders = {};
```

Comment:
I'm not sure why this wasn't here to begin with.  Seems like it would have caused issues for m2m and o2m relations?

Reply:
Yeah, seems right that it should be here.